### ISSUE TITLE
[Publications] Implement revised text size/colors to improve constrast/legibility

### ISSUE THREAD

**Issue by sabrinang** [mozilla/foundation.mozilla.org#6285]
based on https://github.com/mozilla/foundation.mozilla.org/issues/5950#issuecomment-786282143

Areas of improvement are:
- [x] version/date & document size from `#999999` → `#cccccc`
- [x] breadcrumb from `12px` `#b7b9fa` → `16px` `#e7e7fc`
- [x] summary anchor links from `#666666` → `#333333` 
- [x] previous/next from `#666666` → `#333333`

<img width="1230" alt="image" src="https://user-images.githubusercontent.com/2492510/109229708-8a007a00-7778-11eb-8535-af87362c07df.png">

**Comment by jamilasnell** [comment]
Pairing with Daniel

**Comment by mmmavis** [comment]
@sabrinang `#e7e7fc` is not a brand colour. Do we wanna make it "official" by adding it to the style guide? If so, could you provide a name for this new colour? Thanks!

**Comment by sabrinang** [comment]
Yes it would be good to add.
<img width="1243" alt="image" src="https://user-images.githubusercontent.com/2492510/110854457-da440580-8269-11eb-9eb5-eea39d681c2f.png">
I have to think about the naming convention but its two tints lighter than light blue we have _(the pni lilac is actual a tint in between)_. I don't want it to be a blocker @mmmavis if you have any suggestions to try? or we could open up another ticket for colour names.

**Comment by mmmavis** [comment]
Ummmm I can make the codebase allow this one-off colour for now. 

I will file a new ticket for cleaning up all custom colours in the codebase next sprint. There are at least 6 one-off colours now, 5 in [article-page.scss](https://github.com/mozilla/foundation.mozilla.org/blob/e62d219a68e89ea5252439a32d4d2cb0866b8f51/source/sass/components/article-page.scss) and 1 in [publication-page.scss](https://github.com/mozilla/foundation.mozilla.org/blob/190935dc63194473ea3f96343bc1d678d8a33d1b/source/sass/components/publication-page.scss)  (search for `stylelint-disable`). We should really sort these out as it's violating our brand guide and making the codebase harder to maintain.

**Comment by sabrinang** [comment]
Yes thank you! I think you also filed https://github.com/mozilla/foundation.mozilla.org/issues/5906 before but has been stretch for a while so I haven't got to it

**Comment by mmmavis** [comment]
@sabrinang omg I was trying to look for that ticket but GitHub didn't return me anything when I searched for "custom" 🤔 .

I'm gonna close the new ticket I just created and update #5906  with more info then.

**Comment by mmmavis** [comment]
@sabrinang for breadcrumb -- do we want it to appear 16px for all screen sizes? Also wondering what line height we should set it to.

**Comment by sabrinang** [comment]
@mmmavis Let's keep it the same at `line-height: 1.3333333` and yes we can keep it at 16px for mobile to desktop as well.

**Comment by mmmavis** [comment]
@sabrinang thanks!

@danielfmiranda you can use the `set-text-size` mixin for setting mobile/desktop font size and line-height.
